stories/Containers/PictureCard.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { PictureCard, Paragraph, SmallText } from '../../src';
+import { Dog, Poo } from 'styled-icons/fa-solid';

```

Comment:
Can you import these individually? Importanting from the root increases the build time by 10+ seconds (I wish we used react-icons instead)

Reply:
done